pkg/util/docker/dockerfile/dockerfile.go

```diff
@@ -0,0 +1,55 @@
+package dockerfile
```

Comment:
> also does it need to be in a new package?

I want to clean up what we have there right now.
This is a WIP, I'm open to arbitrary changes, including where to place the code.

This code is generic and seemed to fit into the description of `pkg/util`:
```go
// Package util implements various utility functions used in both testing and
// implementation of OpenShift. Package util may not depend on any other
// package in the OpenShift package tree.
```